CircuitPy/FC_Board/lib/pysquared.py

```diff
@@ -145,10 +145,10 @@ def __init__(self):
 
         # Initialize Accelerometer
```

Comment:
The intended use of the lsm303 is actually for its onboard magnetometer not the accelerometer. This should be changed around to instead return mag values rather than redundant access values.